components/inputs/test/input-checkbox.axe.js

```diff
@@ -5,17 +5,22 @@ describe('d2l-input-checkbox', () => {
 
 	it('unchecked', async() => {
 		const elem = await fixture(html`<d2l-input-checkbox aria-label="basic"></d2l-input-checkbox>`);
-		await expect(elem).to.be.accessible;
+		await expect(elem).to.be.accessible();
 	});
 
 	it('checked', async() => {
-		const elem = await fixture(html`<d2l-input-checkbox checked></d2l-input-checkbox>`);
-		await expect(elem).to.be.accessible;
+		const elem = await fixture(html`<d2l-input-checkbox aria-label="basic" checked></d2l-input-checkbox>`);
+		await expect(elem).to.be.accessible();
+	});
+
+	it('is accessible when provided a label', async() => {
```

Comment:
I added a new test here as I believe its valid to provide a label through a slot or the `aria-label` attribute